lib/cmd/compile/scripts.js

```diff
@@ -60,7 +60,8 @@ const _ = require('lodash'),
       [
         require('@babel/preset-env'),
         {
-          targets: Object.assign(helpers.getConfigFileOrBrowsersList('babelTargets'), {})
+          targets: Object.assign(helpers.getConfigFileOrBrowsersList('babelTargets'), {}),
+          ...helpers.getConfigFileValue('babelPresetEnvOptions'),
```

Comment:
this works if `babelPresetEnvOptions` is undefined?

Reply:
ya, object destructuring and spread is good about that, e.g.
```js
> x = null
null
> y = undefined
undefined
> z = {...x, ...y}
{}
>
```

no errors, just nothing gets assigned